### PR TITLE
Fix DOM and React warnings

### PR DESCRIPTION
**Pull request by noisekit**

### CODE REVIEW

File: v3/lib/usePoolData/usePoolData.tsx [hunk 1]
```diff
@@ -120,10 +120,10 @@ export const usePoolData = (poolId?: string) => {
       if (!network.name || !poolId) throw Error('OMG!');
       const poolData = await getPoolData(network.name, poolId);
       if (!poolData.data.pool) {
-        return undefined;
+        throw Error(`Pool ${poolId} not found`);
       }
       return poolData.data.pool;
     },
-    enabled: Boolean(network.isSupported && poolId),
+    enabled: Boolean(network.isSupported && poolId && parseInt(poolId) > 0),
```
[the remark this answers is not in the document]
Comment (jmzwar) [reply v3/lib/usePoolData/usePoolData.tsx:127]: argh

File: v3/ui/src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -160,12 +160,9 @@ export function HomeUi({
             </Thead>
             <Tbody sx={{ tr: { borderBottomColor: 'gray.900', borderBottomWidth: '1px' } }}>
               {preferredPool && collateralTypes ? (
-                <>
-                  {collateralTypes.map((c) => (
-                    <VaultRow key={c.tokenAddress} collateralType={c} poolId={preferredPool.id} />
-                  ))}
-                  <Divider />
```
Comment (jmzwar) [inline v3/ui/src/pages/Home/Home.tsx:167]: We don't need the divider?

**Review by jmzwar — COMMENTED** [review]
Just the one question

File: v3/lib/usePoolData/usePoolData.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (bachstatter) [reply v3/lib/usePoolData/usePoolData.tsx:127]: Nice catch.. Something change on the deployment ( and the reason this mistake was made) preferred pool use to be 1. Pool 0 was suppose to be a special pool for loans without market exposure and the spartan pool (set as preferred pool) was 1.. Either way we need this fix

**Review by bachstatter — APPROVED** [review]
JZ's divider question valid, otherwise looks good

**Pull request merged**